pages/instructorGradebook/instructorGradebook.js

```diff
@@ -28,6 +28,20 @@ router.get('/', function(req, res, next) {
             if (ERR(err, next)) return;
 
             res.locals.user_scores = result.rows;
+            res.locals.user_scores_data = _.map(result.rows, function(row) {
+                var scores = {
+                    user_id: row.user_id,
+                    uid: row.uid,
+                    uin: row.uin ?? '',
+                    user_name: row.user_name ?? '',
+                    role: row.role,
+                };
+                row.scores.forEach(function(score) {
+                    scores[`score_${score.assessment_id}`] = score.score_perc;
+                    scores[`score_${score.assessment_id}_ai`] = score.assessment_instance_id;
```

Comment:
A few other places in the code have used `ai_id` as an abbreviation for `assessment_instance_id`. It's not an essential change, however.